src/ray/gcs/gcs_server/test/gcs_monitor_server_test.cc

```diff
@@ -72,7 +72,7 @@ std::shared_ptr<gcs::GcsPlacementGroup> ConstructPlacementGroupDemand(
 class GcsMonitorServerTest : public ::testing::Test {
  public:
   GcsMonitorServerTest()
-      : mock_node_manager_(std::make_shared<gcs::MockGcsNodeManager>()),
+      : mock_node_manager_(gcs::MockGcsNodeManager()),
```

Comment:
dangling pointer?

Reply:
I think here it should be a copy constructor?

Reply:
Oh right. It's not a reference.